nftnl/src/expr/ct.rs

```diff
@@ -45,6 +46,9 @@ impl Conntrack {
             Conntrack::State => libc::NFT_CT_STATE as u32,
             Conntrack::Status => libc::NFT_CT_STATUS as u32,
             Conntrack::Mark { .. } => libc::NFT_CT_MARK as u32,
+            // TODO: Update this once libc has definitions for NFT_CT_ZONE
+            // (https://github.com/rust-lang/libc/issues/3566)
+            Conntrack::Zone { .. } => 17 as u32,
```

Comment:
Cool. Can you submit a PR to `libc`? We won't merge this with a hardcoded constant.